pkg/build/client/internalversion/buildlogs.go

```diff
@@ -0,0 +1,32 @@
+package internalversion
+
+import (
+	rest "k8s.io/client-go/rest"
+	kapi "k8s.io/kubernetes/pkg/api"
+
+	buildapi "github.com/openshift/origin/pkg/build/apis/build"
+	buildinternalversion "github.com/openshift/origin/pkg/build/generated/internalclientset/typed/build/internalversion"
```

Comment:
I want to avoid this dependency so that I can use this with the openshift/client-go library.